Ruddarr/Views/Movies/MovieDetails+Overview.swift

```diff
@@ -24,7 +24,7 @@ extension MovieDetails {
                 detailsSubtitle
                     .padding(.bottom, 6)
 
-                MovieRatings(movie: movie)
+                MovieRatings(movie: movie).font(.callout).foregroundStyle(.secondary)
```

Comment:
Add font styling here since font styling was removed from `MovieRatings` in order to support different font styling for the list layout and the details screen.